OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -284,4 +295,4 @@ cd "$(SolutionDir)"</PostBuildEvent>
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
+</Project>
```

Comment:
Sure. I'll tell my IDE and git client that they've been bad and shouldn't try to fix stuff someone screwed up long ago, because that's how things are done around here.